.github/actions/run-integration-tests/action.yml

```diff
@@ -61,29 +40,6 @@ runs:
         trap 'cleanup SIGTERM' SIGTERM
 
         echo "IntegrationTestsJob: Start"
-        TESTS_DOCKER_ARGS="-e KUBECONFIG=/root/.kube/config -v /home/runner/.kube:/root/.kube -v /home/runner/work/_temp/.minikube:/home/runner/work/_temp/.minikube" \
-            make -C integration build test &
+            make KUBERNETES_DISTRIBUTION=${KUBERNETES_DISTRIBUTION} integration-tests &
         wait $!
```

Comment:
Maybe we can add a comment with a link to http://mywiki.wooledge.org/SignalTrap#When_is_the_signal_handled.3F to avoid confusion later on on this.